tests/test_tools.py

```diff
@@ -0,0 +1,38 @@
+from symbol_exporter.tools import find_version_ranges
+
+
+def test_all_versions_acceptable():
+    assert find_version_ranges(["1.0", "1.1", "1.1.1"], ["1.0", "1.1", "1.1.1"]) == ">=1.0,<=1.1.1"
```

Comment:
Does this want to fail closed or fail open? i.e. when `1.2` is published should it be incorrectly marked as incompatible or should it risk being marked as compatible?

For now I'd be inclined to fail open and make the upper limit be skipped or default to semver, i.e. `>=1.0` or `">=1.0,<2.0"`.